src/List.mo

```diff
@@ -461,9 +461,9 @@ module {
   /// ) // => "123"
   /// ```
   ///
-  /// Runtime: O(size)
+  /// Runtime: O(size(list))
```

Comment:
This notation is unknown to me. `size(list)` evaluates to as number and thus it becomes equivalent to `O(1)`. Maybe we should just stick to the established `O(n)`, which is universally understood.

Reply:
I'm trying to suggest that its related to the size of the list argument (some functions have two). @kentosugama, any guidance here?

Reply:
Like how would you describe the complexity of append. It's 0(n) where n is the size of the first argument, right?

Reply:
`size(list)` is a function of `list`, so I would argue that that is not constant (with respect to different lists). 

So, I used the notation of `O(size)` in other documentation where the module had functions that would take in two data structures, and it would be unclear how to use the `O(n)` notation, since there would be two `n`s. I preferred something like `O(size1 + size2)` instead of `O(n + m)` because it seemed more clear what the variables are. And to keep the runtime notation consistent for functions that only takes in a single data structure, I thought using `O(size)` was better than using `O(n)`.